### PR TITLE
🐛 frame.html: generate the script tag securely

### PR DESCRIPTION
**Pull request by powerivq**
In order to remove the possibility of generating arbitrary HTML code using the URL argument.

### CODE REVIEW

**Review by erwinmombay — APPROVED** [review]

File: 3p/frame.max.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
 try {
   url = JSON.parse(window.name).bootstrap;
 } catch (e) {}
-document.write('<script src="' + url + '"><' + '/script>');
+var script = document.createElement('script');
+script.src = url;
```
Comment (erwinmombay) [inline 3p/frame.max.html:11]: if we are trying to make this more "secure" would adding trusted types (if supported) help?

Comment (powerivq) [reply 3p/frame.max.html:11]: I thought about it. The list of URLs here will need to be very permissible. It needs to allow many 3p domains that hosts the JS which AMP has no control of. We will certainly not be worse off but the benefits are not huge. Is there a mandate for it?

Comment (erwinmombay) [reply 3p/frame.max.html:11]: there is from security, but lets skip it for this file for now since i dont know if it applies

Comment (erwinmombay) [reply 3p/frame.max.html:11]: LGTM

**Pull request merged**